### ISSUE TITLE
Need to integrate vagrant-registration proxy variables with OpenShift proxy variables

### ISSUE THREAD

**Issue by LalatenduMohanty** [projectatomic/adb-atomic-developer-bundle#488]
The current proxy variables in Red Hat/OpenShift Vagrantfile looks as below

https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/components/rhel/rhel-ose/Vagrantfile#L15

```
#PROXY = "<proxy_server>:<port>"
#PROXY_USER = "foo"
#PROXY_PASSWORD = "mysecretpass"
```

And vagrant-registration needs the information as below

```
  # Specify a HTTP proxy to use. This config option if of the format [<host>|<ip>:<port>], eg mongo:8080
  config.registration.proxy

  # Specify a username to use with an authenticated HTTP proxy
  config.registration.proxyUser

  # Specify a password to use with an authenticated HTTP proxy
  config.registration.proxyPassword
```

IMO we should unify both proxy entries in to one set of entries

**Comment by LalatenduMohanty** [comment]
The possible solution I can think of is using ENV['<>']

```

config.registration.proxy = ENV['PROXY']
config.registration.proxyUser = ENV['PROXY_USER']
config.registration.proxyPassword = ENV['PROXY_PASSWORD']
```

```
PROXY = ENV['PROXY'']
PROXY_USER = ENV['PROXY_USER']
PROXY_PASSWORD = ENV['PROXY_PASSWORD']
```

**Comment by praveenkumar** [comment]
Fixed by PR #493
